Assets/Scripts/UI/InGameUI/MenuLeft/MenuLeft.cs

```diff
@@ -19,23 +20,18 @@ public class MenuLeft : MonoBehaviour
     // Use this for initialization
     public void Start()
     {
-        parent = this.gameObject.transform;
+        parent = gameObject.transform;
 
-        AddMenu("ConstructionMenu", "ConstructionMenu", typeof(ConstructionMenu));
-        AddMenu("OrderMenu", "ConstructionMenu", typeof(OrderMenu));
+        GameObject constructionMenu = AddMenu("ConstructionMenu", "ConstructionMenu", typeof(ConstructionMenu));
+        GameObject orderMenu = AddMenu("OrderMenu", "ConstructionMenu", typeof(OrderMenu));
 
-        GameMenuManager.Instance.AddMenuItem("menu_construction", OnButtonConstruction, 0);
-        GameMenuManager.Instance.AddMenuItem("menu_orders", OnButtonOrder, 1);
+        AddMenuButton(constructionMenu, "menu_construction", 0);
+        AddMenuButton(orderMenu, "menu_orders", 1);
     }
 
-    public void OpenMenu(string menuName)
+    public void OpenCurrentlyMenu()
```

Comment:
OpenCurrentMenu might be a better choice of names. We can discuss it in chat, but this isn't a very clear name.